pkgs/games/xivlauncher/default.nix

```
@@ -13,7 +13,7 @@ in
       owner = "goatcorp";
       repo = "XIVLauncher.Core";
       inherit rev;
-      sha256 = "DlSMxIbgzL5cy+A5nm7ZaA2A0TdINtq2GHW27uxORKI=";
+      sha256 = "aQVfW6Ef8X6L6hBEOCY/Py5tEyorXqtOO3v70mD7efA=";
```

Why is the bare base-64 hash used? This would probably be better off using SRI:

```suggestion
      hash = "sha256-aQVfW6Ef8X6L6hBEOCY/Py5tEyorXqtOO3v70mD7efA=";
```

---

Was just keeping consistent with the previous file. updated!

---

("nix-prefetch-github prints the bare base64 hash" is why it was like that originally, fwiw. maybe i'll try and get a PR upstream to either make the hash used configurable or to just include the `sha256-` prefix by default)

---

Ah, makes sense. Not really sure if it's worth changing then, but doesn't really matter either way.